### PR TITLE
LPS-126294

### PR DESCRIPTION
**Pull request by Odrakir1**

### CODE REVIEW

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsetsStructure.testcase [hunk 1]
```diff
@@ -198,12 +198,59 @@ definition {
...
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Documents and Media");
+
+		AppBuilderAdmin.switchTab(tabName = "Document Types");
+
+		DMDocumentType.addWithoutSave(
+			dmDocumentTypeFieldNames = "Numeric",
+			dmDocumentTypeName = "Document Type Name");
```
Comment (alexcandidos) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsetsStructure.testcase:215]: Hey @Odrakir1 , this test should be made under Web Content / Stuctures and not in Documents and Media / Document Type.

File: portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro [hunk 2]
```diff
@@ -1058,7 +1058,18 @@ definition {
 				value1 = "${fieldPlaceholder}");
 		}
 
-		if (isSet(fieldHelp)) {
+		if (isSet(fieldsetModal) && isSet(fieldHelpTextEdit)) {
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro:1061]: hey @Odrakir1 , could you please change the logic to:

```
if(isSet(fieldHelp)){
if(isSet(fieldsetModal)){
-> do what you implemented ...
}else{
-> do the existing implementation of (isSet(fieldHelp))
}
```

Also, don't use/create a different variable (`fieldHelpTextEdit`) to the same purpose of `fieldHelp`.

**Comment by alexcandidos** [comment]
Hey @Odrakir1 , could you please take a look in my comments? Thanks!

**Comment by Odrakir1** [comment]
@alexcandidos, sorry for that. I'm gonna fix it today and make a new PR.

**Pull request closed without merging**